docs/build-ansible.rst

```diff
@@ -21,8 +21,8 @@ antsibull uses the ``poetry`` tool to build and install::
     # Install poetry
     python3 -m pip install poetry
 
-    # Clone the antisbull repo.
-    git clone https://github.com/ansible-comunity/antsibull
+    # Clone the antsibull repo.
+    git clone https://github.com/ansible-community/antsibull.git
```

Comment:
It does work without the `.git` extension btw. But since GitHub recommends it, let's keep it :)